hubblestack/files/hubblestack_nova/vulners_scanner.py

```diff
@@ -114,11 +133,11 @@ def _process_vulners(vulners):
     :return: A list of dictionaries as hubble.py swallows
     '''
 
-    packages = vulners.get('data', {}).get('packages')
+    packages = vulners.get('packages')
     if not packages:
         return []
 
-    return [{'tag': pkg,
+    return [{'tag': 'Vulnerable package: {0}'.format(pkg),
```

Comment:
The existing implementation reports secure packages like `'Secure packages': '462 out of 462'` but reports vulnerable packages simply using their package name  `{ 'firefox-60.2.2-1.el7.centos.x86_64': u'CESA-2019:0219, CESA-2018:3005, CESA-2018:3833'}`. 

This change aligns the two formats so that you can identify that the failure is because of a vulnerable package specifically (e.g. `{'Vulnerable package: firefox-60.2.2-1.el7.centos.x86_64': u'CESA-2019:0219, CESA-2018:3005, CESA-2018:3833'}`). In our setup, we are processing these failure reports downstream, so its very useful to include the string `"Vulnerable package"` so we can know _why_ the error was reported at all, instead of just having a package name.